### PR TITLE
added CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Sahajj**
Added Code_of_Conduct to the project

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at {{ email }}. All
```
Comment (exitflynn) [inline CODE_OF_CONDUCT.md]: yep you can include my email `tyagiakshit833@gmail.com`

**Review by exitflynn — COMMENTED** [review]
provided email to add

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Sahajj) [reply CODE_OF_CONDUCT.md]: DONE

**Comment by exitflynn** [comment]
perfect, this closes #8

**Pull request merged**